esp/templates/inclusion/qsd/render_qsd.html

```diff
@@ -1,12 +1,22 @@
 {% load markup %}
 
 <div class="qsd_bits hidden">
-	<div class="qsd_header qsd_bits hidden" onclick="qsd_inline_edit('{{ qsdrec.url }}', '{{ qsdrec.edit_id }}');">This is editable text.  Click here to edit the text; click outside the box to save changes.</div>
-	<div class="hidden qsd_bits qsd_default_hidden" id="inline_edit_{{ qsdrec.edit_id }}">
-		<textarea rows="8" cols="80" id="qsd_content_{{ qsdrec.edit_id }}" onblur="qsd_inline_upload('{{ qsdrec.url }}', '{{ qsdrec.edit_id }}');" name="qsd_content" class="qsd_editor qsd_fullsize qsd_bits hidden">{% autoescape on %}{{ qsdrec.content }}{% endautoescape %}</textarea>
-	</div>
+    <div class="qsd_header qsd_bits hidden" onclick="qsd_inline_edit('{{ qsdrec.url }}', '{{ qsdrec.edit_id }}');">
+        {% if qsdrec.id %}
+            This is editable text.  Click here to edit the text; click outside the box to save changes.
+        {% else %}
+            This is a placeholder for editable text that has not yet been added.  Click here to edit the text; click outside the box to save changes.
+        {% endif %}
+    </div>
+    <div class="hidden" id="inline_edit_{{ qsdrec.edit_id }}">
+        <textarea rows="8" cols="80" id="qsd_content_{{ qsdrec.edit_id }}"
+                  onblur="qsd_inline_upload('{{ qsdrec.url }}', '{{ qsdrec.edit_id }}');"
+                  class="qsd_editor {% if not inline %}qsd_fullsize{% endif %} qsd_bits hidden {% if inline and not qsdrec.id %}qsd_halfsize{% endif %}"
```

Comment:
Yeah, I don't know what half the classes do, but there are more changes I wanted to make to the QSD (mostly changing the blur-to-submit behavior, which makes it really easy to accidentally submit changes) and I can clean this up once I get more familiar from that.